lms/templates/credit_notifications/credit_eligibility_email.html

```diff
@@ -30,10 +30,12 @@
             </p>
 
             <p>
-              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your edX course! Click {link} to get your credit now.").format(
-                link=u'<a href="{dashboard_url}">here</a>'.format(
+              ${_(u"Congratulations! You are eligible to receive course credit for successfully completing your {platform_name} course! Click {link_start}here{link_end} to get your credit now.").format(
```

Comment:
I am assuming this text went through a marketing review and I'm loathe to change the strings.